S/Soup3/build_tarballs.jl

```diff
@@ -0,0 +1,65 @@
+using BinaryBuilder
+
+name = "Soup3"
+version = v"3.2.1"
+
+# Collection of sources required to build Soup
+sources = [
+    ArchiveSource("https://download.gnome.org/sources/libsoup/$(version.major).$(version.minor)/libsoup-$(version).tar.xz",
+        "b1eb3d2c3be49fbbd051a71f6532c9626bcecea69783190690cd7e4dfdf28f29"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libsoup-*/
+install_license COPYING
+
+mkdir build_glib && cd build_glib
+meson --cross-file="${MESON_TARGET_TOOLCHAIN}" \
+    --buildtype=release \
+    -Dtls_check=false \
+    -Dtests=false \
+    -Ddocs=disabled \
+    -Dsysprof=disabled \
+    ..
+
+if [[ "${target}" != *darwin* ]]; then
+    # https://github.com/rbgirshick/py-faster-rcnn/issues/706
+    sed -i "s/-R/-Wl,-rpath=/" build.ninja
+fi
+
+if [[ "${target}" != *darwin* && "${target}" != *w64* ]]; then
+    # https://stackoverflow.com/questions/2418157/c-error-undefined-reference-to-clock-gettime-and-clock-settime
+    sed -i "s/\$ARGS -o \$out \$in \$LINK_ARGS/\$ARGS -o \$out \$in \$LINK_ARGS -lrt/" build.ninja
```

Comment:
If you use single quotes you don't have to escape every single dollar sign, and this becomes much more readable:
```suggestion
    sed -i 's/$ARGS -o $out $in $LINK_ARGS/$ARGS -o $out $in $LINK_ARGS -lrt/' build.ninja
```

Reply:
I'm pretty sure variable interpolation doesn't work inside single quotes, though it's been a while

Reply:
That's the point.

Reply:
Oh wait, I misread that line. Thanks